src/azul/service/elasticsearch_service.py

```diff
@@ -502,8 +521,8 @@ def transform_request(self,
             hits = self.plugin.translate_fields(hits, forward=False)
 
             facets = es_response_dict['aggregations'] if 'aggregations' in es_response_dict else {}
-
-            paging = self._generate_paging_dict(es_response_dict, pagination)
+            sort_facet = inverse_translation[pagination['sort']]
```

Comment:
I'm a bit worried how the `sort` field flows through this. It seem that it is reverse translated here just so it can be injected into the link. Four lines down it's reverse translated a second time. This feels convoluted but I don't have a clear vision of how it can be cleaned up. I'd be happy to join a paired programming session to figure it out.